modules/gcp/gke/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "project" {
+  description = "The project ID to host the cluster in (required)"
+  type = "string"
+}
+
```

Comment:
https://github.com/cloudkite-io/terraform-modules/blob/master/modules/gcp/gke/main.tf#L2

We already use the project in a var. Either use that value and make it explicitly called out as a required map in the README or switch all references to this. But don't do both.